src/components/cart/Cart.js

```diff
@@ -19,9 +22,9 @@ import { sumOfValues, getTotalAmount } from "./helpers/cart.general";
 import "./cart.css";
 
 function Cart(props) {
+    const foodList = useSelector((store)=> store.foodList)
```

Comment:
call it as state rather than store, and keep this logic in a function and pass its reference